languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10085,6 +10085,17 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_AVANT_OU_ARRIERE" name="adj avant ou arrière" default="temp_off">
+      <pattern>
+             <token postag="N.*" postag_regexp="yes"/>
+    <marker>
+        <token regexp="yes">avants|arrières</token>
+    </marker>
+</pattern>
+<message>Une erreur d’orthographe semble être détectée.</message>
```

Comment:
Could you make the message a little more explanatory? Something like "Quand 'arrière' est utilisé en tant qu'adjectif, il est invariable.".

Reply:
That's done.